frontend-react/src/components/Table/ColumnData.tsx

```diff
@@ -100,12 +101,9 @@ export const ColumnData = ({
             return action();
         };
         return tableData(
-            <button
-                className="usa-link bg-transparent border-transparent"
-                onClick={() => doAction()}
-            >
+            <Button type="button" unstyled onClick={() => doAction()}>
```

Comment:
Could this be simplified to `onClick={ doAction }` or does TypeScript complain?

Reply:
Yep, worked fine!